components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -159,7 +159,11 @@ class SHOCMacrophysics : public scream::AtmosphereProcess
 
       const int nlev_v = (nlev-1)/Spack::n;
       const int nlev_p = (nlev-1)%Spack::n;
+      const auto& exner_int = PF::exner_function(p_int(i,nlevi_v)[nlevi_p]);
+      const auto& inv_exner_int_surf = 1/exner_int;
+
       wpthlp_sfc(i) = surf_sens_flux(i)/(cpair*rrho_i(i,nlev_v)[nlev_p]);
```

Comment:
Should all the below `*_sfc(i) =` lines be changed to use `(nlevi_v)[nlevi_p]`?
```suggestion
      wpthlp_sfc(i) = surf_sens_flux(i)/(cpair*rrho_i(i,nlevi_v)[nlevi_p]);
```